packages/oas/src/index.ts

```diff
@@ -758,6 +758,20 @@ export default class Oas {
       });
     });
 
+    Object.entries(this.getWebhooks()).forEach(([path, webhooks]) => {
```

Comment:
Can you remove the "this method doesn't support webhooks` comment in the docblock of this method?

Reply:
done!